ts/packages.ts

```diff
@@ -461,21 +461,16 @@ function clickInstallTab(platform: Platform): void {
     installCode.setAttribute('readonly', 'false');
     let windowsTab = document.getElementById('install-tab-windows');
     let unixTab = document.getElementById('install-tab-unix');
-    switch (platform) {
```

Comment:
Given that we use `windows`, `mac` and `linux` as our platforms, do we want to think about changing the definition of the `Platform` type to `'windows' | 'mac' | 'linux'`?

Reply:
yeah sounds good, updated